critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -56,6 +56,7 @@
     {% block license_agreement_input %} {% endblock %}
   </form>
   {% block buttons %} {% endblock %}
+  <br /><small class="text-warning"><em>{{ _('Note: Drafts are not private!') }}</em></small>
```

Comment:
yeah i realized this problem a bit late :/  so should I submit another pull request? I can do this for the other pull requests too, perhaps making everything look a lot more cleaner? Either way, I will note this in future. Thank you